submissions/nauvian-perspective-bentham/nauvian-perspective-bentham.md

```diff
@@ -0,0 +1,46 @@
+### Nauvian Perspective: Bentham and Xisumavoid <author>Bobby</author>
```

Comment:
Okay. So the reason that I don't have any direct text quotes from Bentham currently is that they are substituted by the audio clips. I don't ask you to embed the audio clips if you don't wish to, however I do at least wish that you allow the link to go to a page where the audio clip can be listened to. I can't realistically replace the audio clips with direct quotes otherwise there would be big chunks of text or quotes condensed down that defeats the whole point of them. **I don't think this article can work at all, if supporting audio clips are not able to be included.**

Opinions - yes, I've got too many of my own. I'm trying to remove many of them where I can. Please let me know if you see anymore that probably shouldn't be there :)

Reply:
Conor is currently looking into it if it's possible to somehow host the audio, even if it opens into a new tab. It would be really nice. So we won't remove audio if we can help it. :) But, we would like it if there were Bentham quotes as well - like you did very well for Xisuma.